stacker/providers/aws/default.py

```diff
@@ -106,18 +111,54 @@ def cloudformation(self):
 
         return self._cloudformation
 
-    def get_stack(self, stack_name, **kwargs):
-        try:
-            return retry_on_throttling(
-                self.cloudformation.describe_stacks,
-                kwargs=dict(StackName=stack_name))['Stacks'][0]
-        except botocore.exceptions.ClientError as e:
-            if "does not exist" not in e.message:
-                raise
+    @property
+    def listener(self):
+        # deals w/ multiprocessing issues w/ sharing ssl conns
+        # see https://github.com/remind101/stacker/issues/196
+        pid = os.getpid()
+        if pid != self._pid or not self._listener:
+            session = get_session(self.region)
+            self._listener = CloudListener(
+                LISTENER_NAME, session=session)
+            self._listener.start()
+
+        return self._listener
+
+    def flush_events(self):
+        logger.info("Purging old queue")
```

Comment:
This seems more like debug info, also could be a bit more descriptive.  Also what happened with using a new SQS queue per run?